### ISSUE TITLE
As a developer I would like to have a fault tolerant application (Load balancing)

### ISSUE THREAD

**Issue by bennett000** [rangle/the-clusternator#69]
As a developer I would like to have multiple instances of my application running in parallel so that my application is fault tolerant.

Services will (eventually) require multiple container agents, and other more complex config that will need to sit behind a load balancer.

**Comment by bennett000** [comment]
This is now related to #113 since we'll be using ELB's to terminate SSL

**Comment by bennett000** [comment]
apps now use an ELB, but there's still only one instance attached to them at the moment.

**Comment by bennett000** [comment]
We _are_ currently using ELB's, and in the context of fault tolerance, this is a duplicate of #58
